autopush/db.py

```diff
@@ -96,15 +98,30 @@
 TRACK_DB_CALLS = False
 DB_CALLS = []
 
-# See https://botocore.readthedocs.io/en/stable/reference/config.html for
-# additional config options
-g_dynamodb = boto3.resource(
-    'dynamodb',
-    config=botocore.config.Config(
-        region_name=os.getenv("AWS_REGION_NAME", "us-east-1")
-    )
-)
-g_client = g_dynamodb.meta.client
+SESSION_ARGS = {}
+MAX_SESSIONS = 10
+
+
+class Sessions(object):
+
+    def __init__(self, conf=SESSION_ARGS):
+        self.pool = deque(maxlen=MAX_SESSIONS)
+        session = boto3.session.Session()
```

Comment:
boto_session might avoid pain in the future as we track various 'session' concepts around.